app/safe/internal/state/secret-queue-k8s-delete.go

```diff
@@ -11,13 +11,16 @@ package state
 import (
 	entity "github.com/shieldworks/aegis/core/entity/data/v1"
 	"github.com/shieldworks/aegis/core/env"
-	"github.com/shieldworks/aegis/core/log"
 )
 
 // The secrets put here are synced with their Kubernetes Secret counterparts.
-var k8sSecretDeleteQueue = make(chan entity.SecretStored, env.SafeSecretBufferSize())
+var k8sSecretDeleteQueue = make(chan entity.SecretStored, env.SafeK8sSecretDeleteBufferSize())
```

Comment:
Now the buffer size of 4 different queues are configurable via env vars.

Before it was a single overarching buffer size.